parasolr/query.py

```diff
@@ -261,6 +318,7 @@ def _clone(self) -> 'SolrQuerySet':
         qs_copy.start = self.start
         qs_copy.stop = self.stop
         qs_copy.highlight_field = self.highlight_field
+        #qs.copy.facet_opts = self.facet_opts
```

Comment:
I see this is commented out and that worries me. It should be handled the same way that other dict and list options are. I can add a line to your tests that shows the problem if that's helpful - it appears to be working right now, but you're setting facets by reference so all versions of the queryset share them.